### DISCUSSION TITLE
Why not use ASP.NET Core Identity instead of implementing custom auth?

### DISCUSSION THREAD

**Discussion by pratikghag1089** [spark-dotnet/framework#40] [Q&A]
Why use custom implantations?
`var newUser = await _usersService.CreateUserAsync(userForm);`

Its prefreable using .net core identity.
`var result = await _userManager.CreateAsync(user, userForRegistration.Password); `

**Comment by sturlath** [comment]
Saw this about the new identy endpoints https://andrewlock.net/should-you-use-the-dotnet-8-identity-api-endpoints/

**Reply by pratikghag1089** [reply]
Those are for new idnetity framework. But for existing implementation, its better to use Identity framework (I don't think that is gonna be reworked). Mostly they are gonna provide abstraction to existing identiy framework.
